--- a/vendor/filament/forms/src/Concerns/InteractsWithForms.php
+++ b/vendor/filament/forms/src/Concerns/InteractsWithForms.php
@@ -486,7 +486,7 @@ trait InteractsWithForms
     /**
      * @deprecated Override the `form()` method to configure the default form.
      */
-    protected function getFormStatePath(): ?string
+    public function getFormStatePath(): ?string
     {
         return null;
     }
